lib/solaar/ui/pair_window.py

```diff
@@ -202,7 +202,7 @@ def create(receiver):
 	assistant.set_role('pair-device')
 
 	page_text = _("If the device is already turned on, turn if off and on again.")
-	if receiver.remaining_pairings():
+	if receiver.remaining_pairings() and receiver.remaining_pairings() >= 0:
```

Comment:
Which value does `receiver.remaining_pairings()` have when the device has unlimited pairings?

Reply:
The values are None when unknown, -1 for unlimited, a non-negative integer for how many remaining.  I've seen values from 0 to 19 for c534.  Some other receivers might go up to 40.